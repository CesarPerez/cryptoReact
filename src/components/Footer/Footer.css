.containerFooter {
  left: 0;
  bottom: 0px;
  right: 0;
  background-color: #efefef;
  color: #898989;
  line-height: 3em;
  padding: 1em;
  position: fixed;
}
