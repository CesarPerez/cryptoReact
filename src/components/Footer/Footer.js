import './Footer.css';

const Footer = () => {
  return (
    <>
      <footer className="containerFooter">
        <div>
          <div>
            <span>Â© 2022 Sisar</span>
          </div>
        </div>
      </footer>
    </>
  );
};

export default Footer;
