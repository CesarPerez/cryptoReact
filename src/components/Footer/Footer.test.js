import renderer from 'react-test-renderer'; // ES6
import Footer from './Footer';

test('Should match snapshot', () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
