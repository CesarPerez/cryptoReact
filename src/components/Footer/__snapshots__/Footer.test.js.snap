// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should match snapshot 1`] = `
<footer
  className="containerFooter"
>
  <div>
    <div>
      <span>
        Â© 2022 Sisar
      </span>
    </div>
  </div>
</footer>
`;
